app/src/main/res/values/strings.xml

```diff
@@ -412,6 +412,10 @@
     <string name="creation_date_reversed">Creation date (reversed)</string>
     <string name="alphabetic">Alphabetic</string>
     <string name="alphabetic_reversed">Alphabetic (reversed)</string>
+    <string name="welcome">Welcome to LibreTube</string>
+    <string name="choose_instance">Please choose an instance first!</string>
+    <string name="choose_instance_long">Please choose a Piped instance to use below. The Piped instance will act as a middle man between you and YouTube. These servers are located at different physical places (indicated with flags). Instances that are close to where you live, are likely faster than others. You can still change it later in the instance settings.</string>
```

Comment:
If we have a native English speaker here, it'd be great if they could quickly take a look here!